package.json

```diff
@@ -109,6 +109,7 @@
     "build:babel:cjs": "cross-env BABEL_ENV=production:cjs lerna exec --stream --ignore \"$(node ./scripts/getGlobPackages.js babel)\" --  babel src --out-dir dist/js",
     "build:babel:esm": "cross-env BABEL_ENV=production:esm lerna exec --stream --ignore \"$(node ./scripts/getGlobPackages.js babel)\" -- babel src --out-dir dist/esm",
     "build:docs": "yarn build && lerna run build --scope=@patternfly/react-docs --stream",
+    "build:prdocs": "yarn build && lerna run pr-build --scope=@patternfly/react-docs --stream",
```

Comment:
is `yarn build` running twice in Travis? I think we already have `yarn build` running once in Travis yml config so it's probably fine to remove it from here?